README.md

```diff
@@ -0,0 +1,20 @@
+tabstospaces
```

Comment:
This should be tabtospace, since that's the repo name.

Reply:
Fixed